addons/tmf_loadouts/loadouts/ussr_vdv_1980.hpp

```diff
@@ -0,0 +1,508 @@
+	secondaryAttachments[] = {"cup_optic_pgo7v2"};
+	magazines[] += {
+		LIST_2("CUP_PG7VL_M")
+	};
+};
+
+class matac : car {
+	displayName = "MAT Ammo Carrier";
+	backpack[] = {"CUP_B_HikingPack_Civ"};
+	backpackItems[] = {
+		LIST_2("CUP_PG7VL_M")
+	};
+};
+
+class mattl : car {
+	displayName = "MAT Team Leader";
+	backpack[] = {"CUP_B_HikingPack_Civ"};
+	items[] += {"ACE_MapTools"};
+	linkedItems[] += {"Binocular"};
+	backpackItems[] = {
+		LIST_2("CUP_PG7VL_M")
+	};
+};
+
+class hatg : car {
+	displayName = "HAT Gunner";
+	backPack[] = {"CUP_B_Metis_Gun_Bag"};
+};
+
+class hatac : car {
+	displayName = "HAT Ammo Carrier";
+	backPack[] = {"CUP_B_Metis_Tripod_Bag"};
+};
+
+class hattl : car {
+	displayName = "HAT Team Leader";
+	backPack[] = {"CUP_B_Metis_Tripod_Bag"};
+	items[] += {"ACE_MapTools"};
+	linkedItems[] = {
+		"ItemMap",
+		"ItemCompass",
+		"ItemWatch",
+		"Binocular"
+	};
+};
+
+class mtrg : car {
+	displayName = "Mortar Gunner";
+	secondaryWeapon[] = {"ace_csw_staticMortarCarry"};
+	backPack[] = {"CUP_B_HikingPack_Civ"};
+	backpackItems[] = {
+		LIST_4("ACE_1Rnd_82mm_Mo_HE"),
+		"ACE_1Rnd_82mm_Mo_Illum",
+		"ACE_1Rnd_82mm_Mo_Smoke"
+	};
+};
+
+class mtrac : car {
+	displayName = "Mortar Ammo Carrier";
+	backPack[] = {"CUP_B_HikingPack_Civ"};
+	backpackItems[] = {
+		LIST_4("ACE_1Rnd_82mm_Mo_HE"),
+		"ACE_1Rnd_82mm_Mo_Illum",
+		"ACE_1Rnd_82mm_Mo_Smoke"
+	};
+};
+
+class mtrtl : car {
+	displayName = "Mortar Team Leader";
+	secondaryWeapon[] = {"ace_csw_carryMortarBaseplate"};
+	items[] = { MTR_GEAR };
+	backPack[] = {"CUP_B_HikingPack_Civ"};
+	backpackItems[] = {
+		LIST_4("ACE_1Rnd_82mm_Mo_HE"),
+		"ACE_1Rnd_82mm_Mo_Illum",
+		"ACE_1Rnd_82mm_Mo_Smoke"
+	};
+
+	linkedItems[] = {
+		"ItemMap",
+		"ItemCompass",
+		"ItemWatch",
+		"Binocular"
+	};
+};
+
+class samg : car {
+	displayName = "AA Missile Specialist";
+	backpack[] = {"CUP_B_HikingPack_Civ"};
+	secondaryWeapon[] = {"CUP_launch_9K32Strela"};
+};
+
+class samag : car {
+	displayName = "AA Assistant Missile Specialist";
+	backpack[] = {"CUP_B_HikingPack_Civ"};
+	secondaryWeapon[] = {"CUP_launch_9K32Strela"};
+	linkedItems[] = {
+		"ItemMap",
+		"ItemCompass",
+		"ItemWatch",
+		"Binocular"
+	};
+};
+
+
+class sn : r {
+	displayName = "Sniper";
+	uniform[] = {"U_B_GhillieSuit"};
+	vest[] = {"V_Chestrig_rgr"};
+	headgear[] = {};
+	goggles[] = {"default"};
+	primaryWeapon[] = {"srifle_LRR_F"};
+	scope[] = {"optic_SOS"};
+	sidearmWeapon[] = {"hgun_Pistol_heavy_01_F"};
+	magazines[] = {
+		LIST_7("7Rnd_408_Mag"),
+		LIST_2("HandGrenade"),
+		LIST_4("11Rnd_45ACP_Mag")
+	};
+};
+
+class sp : g {
+	displayName = "Spotter";
+	uniform[] = {"U_B_GhillieSuit"};
+	vest[] = {"V_Chestrig_rgr"};
+	headgear[] = {};
+	scope[] = {"optic_MRCO"};
+	silencer[] = {"muzzle_snds_h"};
+	items[] += {
+		"ACE_MapTools",
+		"ACE_RangeCard",
+		"ACE_Kestrel4500",
+		"ACE_ATragMX"
+	};
+	linkedItems[] = {
+		"ItemMap",
+		"ItemCompass",
+		"ItemWatch",
+		"Laserdesignator",
+		"ItemGPS"
+	};
+};
```

Comment:
Setup at least some sniper/spotter role, even if they aren't completely authentic. Sniper can simply be marksman with a ghillie suit for all I care.